tests/integration_tests/reports/commands_tests.py

```diff
@@ -1081,41 +1081,6 @@ def test_email_dashboard_report_schedule_force_screenshot(
         assert_log(ReportState.SUCCESS)
 
 
-@pytest.mark.usefixtures(
```

Comment:
This good to go too? (I just wasn't seeing a lot of stuff being removed, so want to confirm this is intentional! :) )

Reply:
Same as the last instance, this was a duplicate method. I believe it was a side effect of the merging into either 2.0 or one of the other major versions.